web-ui/static/js/step-page-poll.js

```diff
@@ -1,3 +1,32 @@
+// Called via jsoo in the generated step-logs.js
+function extractStepsToReproduce(startIndex, finishIndex) {
+  if ( // return if already done
+    !!document.getElementById("build-repro-container") &&
```

Comment:
not sure if the double bang is necessary, isn't the value already truthy/falsy?

Reply:
The double bang is "idiomatic" js for `Boolean(object)`